safe/gui/tools/wizard/step_fc15_hazlayer_origin.py

```diff
@@ -0,0 +1,133 @@
+# coding=utf-8
+"""
+InaSAFE Disaster risk assessment tool by AusAid -**InaSAFE Wizard**
+
+This module provides: Function Centric Wizard Step: Hazard Layer Origin
+
+Contact : ole.moller.nielsen@gmail.com
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+"""
+__author__ = 'qgis@borysjurgiel.pl'
+__revision__ = '$Format:%H$'
+__date__ = '16/03/2016'
+__copyright__ = ('Copyright 2012, Australia Indonesia Facility for '
+                 'Disaster Reduction')
+
+from safe.utilities.i18n import tr
+
+from safe.gui.tools.wizard.wizard_strings import (
+    select_hazard_origin_question,
+    select_hazlayer_from_canvas_question,
+    select_hazlayer_from_browser_question)
+from safe.gui.tools.wizard.wizard_step import get_wizard_step_ui_class
+from safe.gui.tools.wizard.wizard_step import WizardStep
+
+
+FORM_CLASS = get_wizard_step_ui_class(__file__)
+
+
+class StepFcHazLayerOrigin(WizardStep, FORM_CLASS):
```

Comment:
How about "HazardLayerOriginStep" - most people reading the code will not have an idea what Fc means. In general it will be great to have natural language names for classes, modules, methods etc.